qt/aqt/deckoptions.py

```diff
@@ -106,7 +99,7 @@ def display_options_for_deck_id(deck_id: DeckId) -> None:
 
 def display_options_for_deck(deck: DeckDict) -> None:
     if not deck["dyn"]:
-        if KeyboardModifiersPressed().shift or aqt.mw.col.sched_ver() == 1:
+        if not aqt.mw.col.v3_scheduler():
```

Comment:
As some add-ons only expose their functionality in the old deck options screen, we'll need to keep the shift check.